src/cesiumutils.ts

```diff
@@ -392,24 +385,43 @@ export function getValueOrUndefined(prop) {
 }
 
 /**
- * Computes the east-north-up difference between two points.
- * Taken from https://community.cesium.com/t/do-not-move-camera-when-setting-trackedentity/9554/4 to keep the camera centered on target entity
- * @param {Cartesian3} left
- * @param {Cartesian3} right
- * @returns {Cartesian3}
+ * Makes the camera look towards a specific point without changing its location
+ * @param position
+ * @param camera
  */
-export function eastNorthUp(left, right) {
-  const leftCartographic = Cartographic.fromCartesian(left);
-  const rightCartographic = Cartographic.fromCartesian(right);
-  const leftUp = Cartesian3.fromRadians(rightCartographic.longitude, leftCartographic.latitude);
-  const rightDown = Cartesian3.fromRadians(rightCartographic.longitude, rightCartographic.latitude);
-  const isEast = rightCartographic.longitude > leftCartographic.longitude;
-  const isNorth = rightCartographic.latitude > leftCartographic.latitude;
-  return new Cartesian3(
-    Cartesian3.distance(left, leftUp) * (isEast ? 1 : -1),
-    Cartesian3.distance(leftUp, rightDown) * (isNorth ? 1 : -1),
-    rightCartographic.height - leftCartographic.height
+export function lookAtPoint(position: Cartesian3, camera: Camera) {
```

Comment:
Shouldn't we use https://cesium.com/learn/cesiumjs/ref-doc/Camera.html#lookAt instead?

Reply:
no, it works differently